develop/TOOLS/DIS/Developing/VisualStudioSolutions.md

```diff
@@ -11,3 +11,5 @@ This section consists of the following topics:
 - [Developing connectors](xref:Developing_connectors_as_Visual_Studio_solutions)
 
 - [Developing Automation scripts](xref:Developing_Automation_scripts_as_Visual_Studio_solutions)
+
+- [Using the DIS MIB Browser](xref:DisGuides_MibBrowser)
```

Comment:
I've moved the DIS MIB Browser guide to the "Development using DIS" section of the documentation. Creating a separate "guide" section is not necessary as the whole purpose of the DataMiner documentation is to be a guide.

Reply:
I moved it again to be under 'Developing connectors'.